opennms-provision/opennms-detector-simple/src/main/java/org/opennms/netmgt/provision/detector/snmp/SnmpDetector.java

```diff
@@ -95,6 +100,32 @@ public void afterPropertiesSet() throws Exception {
         Assert.notNull(m_agentConfigFactory);
     }
 
+    /**
+     * <p>setIsTable</p>
+     *
+     * @return a {@link java.lang.String} object.
+     */
+    public String getIsTable(String table) {
+        if (this.m_isTable) {
+            return new String("true");
+        } else {
+            return new String("false");
+        }
+    }
+
+    /**
+     * <p>getIsTable</p>
+     *
+     * @param table a {@link java.lang.String} object.
+     */
+    public void setIsTable(String table) {
+        if ("true".equalsIgnoreCase(table)) {
+            this.m_isTable = true;
+        } else {
+            this.m_isTable = false;
+        }
+    }
```

Comment:
Both methods `getIsTable()` and `setIsTable()` do not comply with the Java Bean Naming Convention.
This is very bad if you want to use reflections. Please comply to the Java Beans Naming Convention.
Boolean properties should be named as following:
 * Setters: set***PropertyName***, e.g. `setTable(Boolean)`
 * Getter: is***PropertyName***, e.g. `isTable()`

See http://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter